pkg/runner/runner.go

```diff
@@ -19,10 +19,80 @@
 // Package runner handles the execution entry point
 package runner
 
-import "github.com/fgimenez/snappy-cloud-image/pkg/flags"
+import (
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/fgimenez/snappy-cloud-image/pkg/flags"
+	"github.com/fgimenez/snappy-cloud-image/pkg/image"
+)
+
+// Runner is the main type of the package
+type Runner struct {
+	imgDataOrigin image.Pollster
```

Comment:
It seems that both are synonyms [1] [2], here I trusted google translate :) [3] [4]

[1] http://www.merriam-webster.com/dictionary/pollster
[2] http://www.merriam-webster.com/dictionary/poller 
[3] https://translate.google.es/#en/es/pollster
[4] https://translate.google.es/#en/es/poller